R/gen-namespace-docs.R

```diff
@@ -1967,6 +1967,40 @@ NULL
 #' @export
 NULL
 
+#' fftfreq
+#'
+#' Computes the discrete Fourier Transform sample frequencies for a signal of size `n`.
+#'
+#' @note 
+#' By convention, `fft()` returns positive frequency terms first, followed by the negative
+#' frequencies in reverse order, so that `f[-i]` for all `0 < i ≤n/2`
+#' gives the negative frequency terms. For an FFT of length `n` and with inputs spaced
+#' in length unit `d`, the frequencies are:
+#' `f = [0, 1, ..., (n - 1) // 2, -(n // 2), ..., -1] / (d * n)`
+#' 
+#' @note 
+#' For even lengths, the Nyquist frequency at `f[n/2]` can be thought of as either negative
+#' or positive. `fftfreq()` follows NumPy’s convention of taking it to be negative.
+#'
+#' @param n (integer) – the FFT length
+#' @param d (float, optional) – the sampling length scale. The spacing between individual
+#' samples of the FFT input. The default assumes unit spacing, dividing that result by the
+#' actual spacing gives the result in physical frequency units.
+#' @param out (default: `NULL`) the output tensor
```

Comment:
You can't document an argument that's not in the function signature. In this case, the only way to match pytorch's signature is to write a wrapper function that adds those arguments to the signature. 

```
❯ checking Rd \usage sections ... WARNING
  Undocumented arguments in documentation object 'torch_fft_fftfreq'
    ‘options’
  Documented arguments not in \usage in documentation object 'torch_fft_fftfreq':
    ‘out’ ‘dtype’ ‘layout’ ‘device’ ‘requires_grad’
```

You can the function name to the list here: 

https://github.com/mlverse/torch/blob/main/tools/torchgen/R/r.R#L130

Regenerate the wraper code so the automatically wraped function name starts with a `.` then
You can write the wraper function in this file adding those arguments to the signature and mangling them
to create the correct call to `.torch_fft_fftreq`.

https://github.com/mlverse/torch/blob/main/R/wrapers.R